arangod/Replication2/ReplicatedLog/AgencyLogSpecification.h

```diff
@@ -276,6 +276,7 @@ struct LogCurrentSupervision {
   std::optional<uint64_t> targetVersion;
 
   auto toVelocyPack(VPackBuilder&) const -> void;
+  static auto fromVelocyPack(velocypack::Slice) -> LogCurrentSupervision;
```

Comment:
When I first saw this I thought "huh?",  then I checked; this function exists already, and ultimately ends up being a `deserialize` call; what happens if you just try and remove it?